rpi-eeprom-update

```diff
@@ -230,9 +230,9 @@ rpi-eeprom-update [options]... [FILE]
    A backup of the current EEPROM config file is written to ${FIRMWARE_BACKUP_DIR}
    before applying the update.
 
-   -a Install the latest critical update if necessary.
-   -d Use the default bootloader config instead of migrating the current settings.
-   -f Install the given file instead of the latest critical update.
+   -a Install the latest update if necessary
+   -d Use the default bootloader config instead of migrating the current settings
+   -f Install the given file instead of the latest critical update
```

Comment:
```suggestion
   -f Install the given file instead of the latest ${FIRMWARE_RELEASE_STATUS } update
```

Reply:
The usage message is used to build a man page when the package is installed, so this wouldn't work. I've changed 'critical' to 'applicable' instead.